pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentUploadDownloadRestletResource.java

```diff
@@ -555,6 +723,67 @@ public void uploadSegmentAsMultiPart(FormDataMultiPart multiPart,
     }
   }
 
+  @POST
+  @ManagedAsync
+  @Produces(MediaType.APPLICATION_JSON)
+  @Consumes(MediaType.MULTIPART_FORM_DATA)
+  @Path("/v3/segments")
```

Comment:
Consider making it more specific, such as `/segments/batchUpload`

Reply:
sure, will make this change.